README.md

```diff
@@ -1,15 +1,17 @@
 ## IO 500 in Four Easy Steps
 
+*NOTE: please use the "io500-app" repo for ISC20 and later list submissions*
+
 ```bash
-git clone https://github.com/VI4IO/io-500-dev.git -b io500-sc-19
-cd io-500-dev
-./utilities/prepare.sh
+git clone https://github.com/VI4IO/io500-app.git -b io500-isc20
+cd io500-app
+./prepare.sh
 ./io500.sh
 ```
 
 This is designed to work from an interactive login (e.g. it won't work from a login node) and uses 'mpirun -np 2' to do the simplest possible MPI run.  It runs a trivially sized problem and uses a serial implementation of find.  Hopefully it just works for you; if it doesn't, please let us know.
 
-To improve it, edit i0500.sh one function at a time to grow the problem size and replace the serial find with something much faster (you may want to try the provided python parallel find). You may also need to take whatever steps necessary to run it with a job scheduler. 
+To improve it, edit io500.sh one function at a time to grow the problem size and replace the serial find with something much faster (you may want to try the provided python parallel find). You may also need to take whatever steps necessary to run it with a job scheduler. 
```

Comment:
This comment is out of date.  The values used to be set low.  Now the default values are very high and the stonewall is set low.  To grow it, they just set stonewall to be 300 I think.

Reply:
We could replace with: The default declared workload can be used for the required stonewall of 300 seconds, you can just change the stonewall value.

Reply:
It is true that this comment is out of date, but this issue predates my patch and shouldn't necessarily prevent it from landing.  Even so, people shouldn't really be following these instructions anymore, as all of the tuning is done in the io500-app `config-foo.ini` file, which the "io500.sh" script there inherits, and this one is totally unused.  Only the `utilities/io500_fixed.sh` script is used.